tools/sendresetemail.py

```diff
@@ -1,7 +1,7 @@
-from app import app
+import logging
```

Comment:
Anything under `tools` is intended to be run from the command line; please leave these as `print`s